fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -189,6 +190,12 @@ class FullyShardedDataParallel(nn.Module):
             module to be wrapped with FSDP.
         process_group (Optional):
             process group for sharding
+        process_group_reduce_scatter (Optional):
+            process group for reduce scatter
+            if the process group is specified as ProcessGroupName.all_gather, the reduce_scatter
```

Comment:
This is not very clear TBH. Can you reword this to say 1)what is the default value and what that means 2) what happens if a user passes in a valid process group and 3) what is the rollback option for this feature.

Reply:
why not pass ProcessGroupName.default in case we want to turn off the feature?

Reply:
That should work as well.